Core/clim-basic/extended-streams/text-formatting.lisp

```diff
@@ -50,7 +50,7 @@
                        `(:absolute ,text-margin)
                        `(:relative 0)))
     (thunk :bottom `(:relative 0))
-    (setf (stream-text-margins instance) text-margins)))
+    (setf (slot-value instance 'margins) text-margins)))
```

Comment:
Why not use the accessor?

Reply:
because `(setf stream-text-margins)` has an around method which tries to merge new margins with old ones (and that calls `stream-text-margins` on an the yet-unbound slot)